litmus-portal/frontend/public/locales/en/translation.yaml

```diff
@@ -150,6 +150,15 @@ community:
   statsHeading: 'Litmus users around the world:'
   litmusChaos: Litmuschaos
   follow: Follow
+  feedback:
+    title: What do you think of Litmus?
+    subHeading: Provide your valuable feedback
+  joinSlack: Litmuschaos on
+  communityEvents:
+    title: Know about Community Events
+    subHeading: Explore about monthly community sync ups, Kubernetes meet ups and other events in CNCF
```

Comment:
syncups

Reply:
https://www.urbandictionary.com/define.php?term=Sync%20up Please refer to the English dictionary

Reply:
PTAL at the event name https://www.meetup.com/Kubernetes-Chaos-Engineering-Meetup-Group/events/279316607/ for the sync up it's still fine.